app/helpers/settings_helper.rb

```diff
@@ -0,0 +1,16 @@
+module SettingsHelper
+  def generate_form_field(setting, form)
+    if %w(site.display_login_link site.only_admin_can_upload site.signup_enabled site.header_inverse).include?(setting.var)
+      return form.select :value, [%w(0 0), %w(1 1)], {}, { class: 'form-control' }
+    end
+
+    if %w(site.theme).include?(setting.var)
+      return form.select :value, [%w(default default), %w(dark dark), %w(white white)], {}, { class: 'form-control' }
+    end
+
+    if %w(site.name site.favicon).include?(setting.var)
```

Comment:
__[RuboCop]__

Style/PercentLiteralDelimiters: `%w`-literals should be delimited by `[` and `]`.


<!-- Comment by [SideCI](https://sideci.com) -->